src/NumberField/NumberField.test.tsx

```diff
@@ -14,13 +14,13 @@ const NumberFieldWithState = ({
   valueIn = 100,
   ...rest
 }: {
-  onChange?: (event: ChangeEvent<HTMLInputElement>) => void;
-  valueIn?: number;
+  onChange?: (value: number) => void;
```

Comment:
Is this correct? onChange takes an event. Why do we even need to override the type for this?

Reply:
Hasn't been addressed.